PubSub/local.properties

```diff
@@ -0,0 +1,11 @@
+## This file is automatically generated by Android Studio.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Fri Mar 11 12:42:32 CST 2016
+sdk.dir=/opt/adt-bundle-mac-x86_64-20130917/sdk
```

Comment:
Noticed there's a gitignore entry for ``local.properties`` but maybe this file was committed by accident?